lib/websocket/websocket.go

```diff
@@ -171,6 +178,7 @@ func (h *connections) subscribeConn(connID, tokenString string) error {
 
 	claims, ok := tok.Claims.(*token.Claims)
 	if !ok || claims.Subject == "" {
+		h.removeConn(connID)
```

Comment:
Think this might close up some bad connections we might had as the connection handler creates any connection and gives it an ID but the subscriber doesn't clean it up.